tests/models/test_hparams.py

```diff
@@ -320,8 +320,7 @@ def test_collect_init_arguments(tmpdir, cls):
     if isinstance(model, (SubClassBoringModel, NonSavingSubClassBoringModel)):
         assert model.hparams.subclass_arg == 1200
 
-    if isinstance(model, AggSubClassBoringModel):
-        assert isinstance(model.hparams.my_loss, torch.nn.CosineEmbeddingLoss)
+    assert "my_loss" not in model.hparams
```

Comment:
Does this mean that the PR introduces a BC-breaking change?

Reply:
sort of yes.. I actually wanted to discuss this. since my main motivation was to disable saving the modules only, but losses are also inherited from `nn.Module`. If we agree with this, we can mark this as breaking change.

Reply:
Yeah, that's a big limitation as the loss is often made configurable.